services/cloudinary.js

```diff
@@ -0,0 +1,11 @@
+import {v2 as cloudinary} from 'cloudinary';
+
+
+cloudinary.config({ 
```

Comment:
This is a config file, not a service, move it to another folder named config or something where we can store all our config files